src/extractors/custom/www.newyorker.com/index.js

```diff
@@ -4,15 +4,15 @@
 export const NewYorkerExtractor = {
   domain: 'www.newyorker.com',
   title: {
-    selectors: ['h1.title'],
+    selectors: ['h1[class^="ArticleHeader__hed"]'],
```

Comment:
On layouts like the one on https://www.newyorker.com/humor/daily-shouts/teas-you-should-probably-get-rid-of-already , the article header elements seem to be present twice on the page (the second being hidden using CSS), could you please update the selector to handle such cases?